h/api_client/api_client.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+"""A Python client for the Hypothesis API."""
+import requests
+import pyramid.traversal
+
+
+class APIError(Exception):
+
+    """Raised when a request to the API fails for any reason."""
+
+    pass
+
+
+class ConnectionError(APIError):
+
+    """Raised when a request to the API fails because of a network problem."""
+
+    pass
+
+
+class Timeout(APIError):
+
+    """Raised when a request to the API times out."""
+
+    pass
+
+
+class HypothesisAPIClient(object):
```

Comment:
I think I'd still rather this was just `Client`. We have namespaces in Python, so let's lean on them to make our code easier to read.